third_party/libcoap/repo/scripts/build.sh

```diff
@@ -23,10 +23,10 @@ case "${PLATFORM}" in
              ;;
 esac
 
-err=$?
-if test $err = 0 -a -n "$WITH_TESTS" ; then
+error=$?
```

Comment:
Is modifying things in `third_party/` intentional?

Reply:
Not intentional..